build-in-container/mariner-docker-builder.sh

```diff
@@ -56,12 +56,31 @@ cleanup() {
     docker rmi -f $(docker images -aq --filter reference="mcr.microsoft.com/mariner-container-build")
 }
 
-create_custom_repo_file() {
-    echo -e "\n" > $custom_repo_file
-    for repo_file in $(echo $1 | tr "," "\n")
+copy_custom_repo_file() {
+    RPM_repo_file=
+    mkdir -p $tool_dir/scripts/custom_repos
+    for repo_file in $(echo $1 | tr " " "\n")
     do
-        cat $(realpath $repo_file) >> $custom_repo_file
-        echo -e "\n" >> $custom_repo_file
+        repo_file_name=${repo_file##*/} # remove prefix ending in '/'
+        cp $(realpath $repo_file) $tool_dir/scripts/custom_repos/$repo_file_name # copy repo_file inside container
+        RPM_repo_file+="/mariner/scripts/custom_repos/$repo_file_name,"
+    done
+}
+
+validate_custom_repo_file() {
+    for repo_file in $(echo $1 | tr " " "\n")
+    do
+        # exit if $repo_file doesn't exist
+        echo "repo_file is $repo_file"
+        if [[ ! -f $(realpath $repo_file) ]]; then
+            echo -e "-------- \033[31m ALERT: $repo_file doesn't exist. Exiting\033[0m --------"
+            exit
```

Comment:
nit: If we are going to exit due to an error, we should do `exit 1` or some other non-zero exit code.